src/Parser/VengeanceDemonHunter/CombatLogParser.js

```diff
@@ -170,7 +188,7 @@ class CombatLogParser extends MainCombatLogParser {
       />,
       <StatisticBox
         icon={<Icon icon="ability_druid_naturalperfection" alt="Damage absorbed" />}
-        value={`${formatNumber(this.totalDamageTakenAbsorb / this.fightDuration * 1000)} DAPS`}
+        value={`${formatNumber(this.totalDamageTakenAbsorb / this.fightDuration * 1000)} APS`}
         label='Damage absorbed'
```

Comment:
If I properly deduced from our earlier conversation that absorbed damage is not a special case for DHs, this should be removed. If you wish to reveal the damage absorbed separate from healing done, please do this in the tooltip of the healing done statistic.